src/api/db/structure.sql

```diff
@@ -397,16 +397,11 @@ CREATE TABLE `configurations` (
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
 
-CREATE TABLE `data_migrations` (
```

Comment:
Why was this table removed?

Reply:
@Ana06 I think you should run `rake db:migrate:with_data` to get the data_migrations table back.

Reply:
good question 🤔